src/main/scopes/wc/wc-prefix-map.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const wcPrefixMap = new Map([
+  ['@carbon/web-components/es', 'cds'],
+  ['@carbon/web-components/es-custom', 'cds-custom'],
+  ['@carbon/ibmdotcom-web-components/es', 'c4d'],
+  ['@carbon/ibm-products-web-components/es', 'c4p'],
+  ['@carbon/ibm-products-web-components/es-custom', 'c4p']
+])
```

Comment:
Let's stick with how the other maps are being defined throughout the code, like in [`WcNodeHandlerMap`](https://github.com/ibm-telemetry/telemetry-js/pull/308/files#diff-6cb1399f7fd3237d21ffe52fd906824e7ff7073b7af7b7db01624d39dadf884cR20-R26).